frontend/public/components/nav.jsx

```diff
@@ -233,6 +234,27 @@ const NavSection = connect(navSectionStateToProps)(
 
 const Sep = () => <div className="navigation-container__section__separator" />;
 
+
+const UserNavSectionWrapper = connectToFlags(FLAGS.AUTH_ENABLED, FLAGS.OPENSHIFT)(({flags, closeMenu}) => {
+  if (!flags[FLAGS.AUTH_ENABLED]) {
```

Comment:
This might be overkill, but if you check

```jsx
 if (!flags[FLAGS.AUTH_ENABLED] || flags[FLAGS.OPENSHIFT] === undefined) {
  return null;
}
```

It would prevent the "My Account" menu from ever displaying while we are still doing discovery. The `flags[FLAGS.OPENSHIFT]` will be undefined until the detection is complete.